.github/workflows/publish-js-client.yml

```diff
@@ -16,6 +20,14 @@ on:
           - prepatch
           - preminor
           - premajor
+      npm_org:
+        description: NPM Organization
+        required: true
+        default: solana
+        type: choice
+        options:
+          - solana
+          - solana-program
```

Comment:
Could we not get this information from the `package.json` instead (by parsing the package name) instead? That way there is no room for human mistake?

Reply:
Good idea, I was lazy